src/ttt/board.clj

```diff
@@ -0,0 +1,31 @@
+(ns ttt.board
+  (:require [clojure.math.numeric-tower :as math]))
+
+(defn initial-board
+  ([board-size]
+   (into [] (range (* board-size board-size))))
+  ([]
+   (initial-board 3)))
+
+
+(defn place-marker
+  [board position marker]
+  (assoc board position marker))
+
+(defn position_available?
+  [board position]
+  (= (nth board position) position))
```

Comment:
if some of these functions are private functions, you should make them private using `defn-`

Reply:
Also, let's try and test only the public facing functions and test the logic inside of our private functions through those tests. For instance, your `make-move` function's 2nd test seems to already do this sort of test for an invalid move.